src/client/sandbox/shadow-ui.js

```diff
@@ -293,6 +302,14 @@ export default class ShadowUI extends SandboxBase {
     attach (window) {
         super.attach(window, window.document);
 
+        var document = window.document;
+
+        if (document.head)
```

Comment:
I guess we can create the separate function `_markShadowUIContainers (head, body)` and check element existence incide this.
```js
_markShadowUIContainers (head, body) {
     if (head)
          ShadowUI._markShadowUIContainerCollections(head);

     if (body)
         ShadowUI._markShadowUIContainerCollections(head);
}
```